app/Http/Controllers/Api/LicenseSeatsController.php

```diff
@@ -116,16 +116,20 @@ public function update(Request $request, $licenseId, $seatId)
             return response()->json(Helper::formatStandardApiResponse('success', $licenseSeat, trans('admin/licenses/message.update.success')));
         }
 
+        // the logging functions expect only one "target". if both asset and user are present in the request,
+        // we simply let assets take precedence over users...
+        if ($licenseSeat->isDirty('assigned_to')) {
+            $target = $is_checkin ? $oldUser : User::find($licenseSeat->assigned_to);
+        }
+        if ($licenseSeat->isDirty('asset_id')) {
+            $target = $is_checkin ? $oldAsset : Asset::find($licenseSeat->asset_id);
+        }
+
+        if(is_null($target)){
```

Comment:
Can we fix the cuddling on this? 

`if ($licenseSeat`

vs

`if(is_null($target))`

I'd prefer `if (is_null($target))` with the space

Reply:
Of course, sorry.